dScripts/WhFans.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include "CppScripts.h"
+#include "GameMessages.h"
+#include "EntityManager.h"
+#include "PhantomPhysicsComponent.h"
+
+class WhFans : public CppScripts::Script
+{
+public:
+	void OnStartup(Entity* self);
```

Comment:
All of these methods here need overrides after their declaration.  `OnStartup(Entity* self) override;`

Reply:
Togglefx does not need an override